src/core/raps/utils.ts

```diff
@@ -95,22 +95,27 @@ const getStateDiff = async (
 ): Promise<unknown> => {
   const tokenAddress = quote.sellTokenAddress;
   const fromAddr = quote.from;
-  const { chainId } = await provider.getNetwork();
-  const toAddr =
-    quote.swapType === 'normal'
-      ? getRainbowRouterContractAddress(chainId)
```

Comment:
we need to update this on the app as well - it is not technically wrong (the `allowanceTarget` and the `to` are the same for crosschain swaps), but for the sake of consistency can you keep the CrosschainQuote flow with the allowance target the same, and just replace the `getRainbowRouterContractAddress` with `getTargetAddressForQuote`

Reply:
done